attestations-collections/playground/polygon/factory/hydra-s1-simple-factory-badges.ts

```diff
@@ -293,6 +293,28 @@ export const factoryBadges = [
     },
     links: []
   },
+  {
+    internalCollectionId: 2538036,
+    name: "DAOtest ZK Badge",
+    description: "ZK Badge owned by Sismo testers. This Badge is used for test.",
+    image: "testers.svg",
+    groupGeneratorName: "testers",
+    publicContacts: [
+      {
+        type: "twitter",
+        contact: "@Sismo_eth"
```

Comment:
Hey @xokuzya, can you change this one to put your own twitter account?
Thanks 🙏